### PR TITLE
Enable cppcheck lints in behavior_velocity_planner

### PR DESCRIPTION
**Pull request by nnmm**
I only fixed cppcheck lints and missing includes (for which I wrote a script), as well as clang-tidy lints. There was one about the `stop_point` variable in stop_line/scene.cpp being possibly uninitialized.

### CODE REVIEW

File: planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/include/utilization/interpolation/cubic_spline.hpp [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
- * SOFTWARE.
-*/
+// Copyright 2018 Forrest
```
Comment (jilaada) [inline planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/include/utilization/interpolation/cubic_spline.hpp:1]: This is another example which is the same as the #153 copyright issue.

**Review by jilaada — CHANGES_REQUESTED** [review]
Great changes! Though I needed to make some modifications below in order to get it to work - Those are listed below so once those are done I am happy to merge. 

Just a quick question - running clang-tidy on a random sample of files yields some warnings generated. Are we not fixing the warnings?

**Comment by nnmm** [comment]
@jilaada I made the change to package.xml – I think I had already changed it locally and didn't commit it for some reason.

**Review by jilaada — APPROVED** [review]
@nnmm Great! Thanks for adding that commit. I think for the copyright issue we can fix them in another PR as this is not the only instance

**Pull request merged**